cmd/crc/cmd/config/config.go

```diff
@@ -32,6 +34,7 @@ func RegisterSettings(cfg *config.Config) {
 	cfg.AddSetting(PullSecretFile, "", config.ValidatePath, config.SuccessfullyApplied)
 	cfg.AddSetting(DisableUpdateCheck, false, config.ValidateBool, config.SuccessfullyApplied)
 	cfg.AddSetting(ExperimentalFeatures, false, config.ValidateBool, config.SuccessfullyApplied)
+	cfg.AddSetting(NetworkMode, string(network.DefaultMode), network.ValidateMode, network.SuccessfullyAppliedMode)
```

Comment:
this eventually needs documentation. what are the possible values? what is the default? a short oneliner what it does?